config/schema/metatag_context.schema.yml

```diff
@@ -0,0 +1,32 @@
+metatag.metatag_context.*:
+  type: config_entity
+  label: 'Metatag'
+  mapping:
+    id:
+      type: string
+      label: 'ID'
+    label:
+      type: label
+      label: 'Type'
+    uuid:
+      type: string
+    title:
+      type: string
+      label: 'Page title'
+    description:
+      type: string
+      label: 'Description'
+    abstract:
+      type: string
+      label: 'Abstract'
+    keywords:
```

Comment:
I think the problem here is that we want to make this schema flexible to store other things. I'm not sure we can provide storage for each thing individually. It may have to go into a general 'data' or 'defaults' array.